pkg/ruler/ruler.proto

```diff
@@ -23,7 +23,14 @@ service Ruler {
   rpc Rules(RulesRequest) returns (RulesResponse) {};
 }
 
-message RulesRequest {}
+message RulesRequest {
+  enum RuleType{
+    AnyRule = 0;
+    AlertingRule = 1;
+    RecordingRule = 2;
+ }
+ RuleType filter = 1;
```

Comment:
[nit] Indentation isn't quite right, not sure if this should be picked up by a linter...
```suggestion
  }
  RuleType filter = 1;
```